src/integrated/soil_plant_hydrology_model.jl

```diff
@@ -59,11 +59,10 @@ function SoilPlantHydrologyModel{FT}(;
 ) where {FT, SM <: Soil.AbstractSoilModel{FT}}
 
     # These may be passed in, or set, depending on use scenario.
-    (; atmos, radiation) = land_args
+    (; soil_driver, atmos, radiation) = land_args
```

Comment:
just to be sure I am following, here we still need to pass it in because we need to get the albedo from the user, while in the soil_canopy_model we can set it in the constructor because the albedo is set by the user as part of the soil model?

Reply:
Could we add a note in the docstring for this model to make that clear?